packages/protocol/package.json

```diff
@@ -10,11 +10,17 @@
   "types": "src/index",
   "module": "src/index",
   "scripts": {
-    "clean": "rimraf lib tsconfig.tsbuildinfo",
+    "prepare": "yarn generate",
+    "clean": "rimraf lib tsconfig.tsbuildinfo && yarn generate clean",
```

Comment:
why use a script, which not works on windows, when you already have the solution?
`clean: "rimraf lib tsconfig.tsbuildinfo inscription.json src/data/inscription.ts"`

Reply:
the script has some minor benefits:
- we do not have to duplicate the path to the generated inscriptions.ts
- we can easily override the generation temporary, by overriding the URI param (without modifying any sources)
-